src/main/java/org/gradle/profiler/studio/LauncherConfigurationParser.java

```diff
@@ -30,12 +36,30 @@ public LaunchConfiguration calculate(Path studioInstallDir) {
         List<Path> classPath = Arrays.stream(jvmOptions.string("ClassPath").split(":")).map(s -> FileSystems.getDefault().getPath(s.replace("$APP_PACKAGE", actualInstallDir.toString()))).collect(Collectors.toList());
         String mainClass = jvmOptions.string("MainClass");
         Map<String, String> systemProperties = mapValues(jvmOptions.dict("Properties").toMap(), v -> v.replace("$APP_PACKAGE", actualInstallDir.toString()));
-        Path javaCommand = actualInstallDir.resolve("Contents/jre/jdk/Contents/Home/bin/java");
+        systemProperties.put("gradle.profiler.port", String.valueOf(studioPluginPort));
+        systemProperties.put("idea.trust.all.projects", "true");
+        Path javaCommand = actualInstallDir.resolve("Contents/jre/Contents/Home/bin/java");
```

Comment:
This path has already been fixed on master, so there is a conflict now.

Reply:
rebased